tests/integration/Owncloud/OcisPhpSdk/DriveTest.php

```diff
@@ -72,11 +72,51 @@ public function testDeleteEnabledDrive(): void
 
     public function testGetDriveRole(): void
     {
+        if(getenv('OCIS_VERSION') === "stable") {
+            $this->markTestSkipped();
+        };
         $role = $this->drive->getRoles();
         $this->assertContainsOnlyInstancesOf(
             SharingRole::class,
             $role,
             "Array contains not only 'SharingRole' items"
         );
     }
+
+    public function testCreateDriveInvite(): void
+    {
+        if(getenv('OCIS_VERSION') === "stable") {
+            $this->markTestSkipped();
+        };
```

Comment:
Only stable and latest commits are used in the building the drone. It feels easy to draw env from the drone runtime of stable and master only.

Reply:
yes, but that is not the point. We skip the test on oCIS 5.0.0 but when in real-live s.o. runs the code with 5.0.0 it will crash unexpected. We need to catch that case